index.html

```diff
@@ -254,15 +255,15 @@
 
           <p>If a user has exited pointer lock via the <a>default unlock user
           gesture</a>, or pointer lock has not previously been entered for this
-          document, an event generated as a result of an
-          <a>engagement gesture</a> must be received by the document before
+          shadow-including document, an event generated as a result of an
```

Comment:
I don't believe this is necessary. In general there is no such term as "shadow-including document". It is just "document". You can be "in a shadow-including document" but the "shadow-including" modifies "in", not "document." So any case where there is no "in" seems like a wrong change.